src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs

```diff
@@ -75,15 +77,15 @@ public class Aapt2Link : Aapt2 {
 		AssemblyIdentityMap assemblyMap = new AssemblyIdentityMap ();
 		List<string> tempFiles = new List<string> ();
 
-		public override bool Execute ()
+		public override bool RunTask ()
 		{
 			Yield ();
 			try {
 				var task = this.RunTask (DoExecute);
 
 				task.ContinueWith (Complete);
 
-				base.Execute ();
+				base.RunTask ();
 			} finally {
 				Reacquire ();
 			}
```

Comment:
Since we have `AndroidAsyncTask` now, can we put this yield-try-finally-reacquire shenanigans in there, too?

I think the same pattern is in here a few times -- would be nice to override `RunTask` and that's it.